pyvera/__init__.py

```diff
@@ -119,6 +119,12 @@ def get_devices(self, category_filter=''):
                 self.devices.append(VeraDimmer(item, self))
             elif (item.get('deviceInfo') and
                   item.get('deviceInfo').get('categoryName') ==
+                  'Dimmable Light'):        
+                dimmer = VeraDimmer(item, self)
+                dimmer.category = "Dimmable Switch"
```

Comment:
Rather that changing the category - I think it might be better to get the same effect by overriding `is_dimmable` in VeraDimmer:-

````python
    @property
    def is_dimmable(self):
        """Device is dimmable."""
        return True
````
That way - once we've decided the object is a Dimmer - other checks will be consistent.

For purity also probably better to refactor the base `VeraDevice` class slightly:-
````python
    @property
    def is_dimmable(self):
        """Default device is not dimmable."""
        return False
````

Longer term I think this would be a better pattern for the others objects too - I will refactor along these lines